docs/firebase_in_libraries.md

```diff
@@ -1,76 +0,0 @@
-# Using Firebase from a framework or a library
```

Comment:
I would prefer adding a note explaining why it is not relevant for Firebase 7 instead of deleting. Also, it seems still to be a relevant concern for zip, Carthage and binary distributions where we keep using static binaries.